services/session.go

```diff
@@ -315,7 +318,10 @@ func LoadSessionsFromDisk() error {
 			}
 
 			// Connect PWD daemon to the new network
-			if err := ConnectNetwork("pwd", s.Id); err != nil {
+			if s.PwdIpAddress == "" {
+				log.Fatal("Cannot load stored sessions as they don't have the pwd ip address stored with them")
```

Comment:
This Will fail when we deploy the new version right? Because none of the current sessions Will have that field. Also, is log.Fatal the best thing to do here? Maybe it's best to just avoid this session and remove it from the sessions map